wacruit/src/apps/user/services.py

```diff
@@ -39,3 +43,14 @@ def create_user(self, request: UserCreateUpdateRequest) -> UserCreateResponse:
     def list_users(self) -> list[UserDetailResponse]:
         users = self.user_repository.get_users()
         return [UserDetailResponse.from_orm(user) for user in users]
+
+    def update_invitaion_emails(
+        self, user: User, request: UserUpdateInvitationEmailsRequest
+    ) -> UserDetailResponse:
+        user.github_email = request.github_email
+        user.notion_email = request.notion_email
+        user.slack_email = request.slack_email
```

Comment:
변경되지 않는 이메일의 경우 원래의 이메일 값을 항상 전달된다고 가정한 건가요??
그렇지 않다면 partial update의 경우 null 처리를 해주어야할 것 같은데.. `or` 를 이용해서 null chaining 하거나, 아래와 같이 update를 하는 방식으로요
```suggestion
        user.__dict__.update(request.dict())
```

Reply:
수정했습니다
https://github.com/wafflestudio/wacruit-server/pull/6/commits/a5dbc53818bc3337150e0b33d15cac4c99873ec8